content/docs/releases/release-notes/release-notes-1.14.md

```diff
@@ -57,7 +57,8 @@ so that clients can verify the identity of the metrics server.
 
 The liveness probe of the cert-manager controller Pod is now enabled by default.
 
-There is a new option `.spec.keystores.pkcs12.algorithms` to specify encryption and MAC algorithms for PKCS.
+There is a new option `.spec.keystores.pkcs12.profile` to specify encryption and HMAC algorithms for PKCS keystores.
+See the [API reference](../../../docs/reference/api-docs/#cert-manager.io/v1.PKCS12Profile) for configuration options.
```

Comment:
I think you need to link to the markdown document. See:
 * https://github.com/cert-manager/website/blob/f91577c1d673dc6cc599065d9daeb0b07667ec20/content/docs/configuration/acme/http01/README.md?plain=1#L51-L53